### ISSUE TITLE
npm package file `i18next/dist/es/defaults.js` throws SyntaxError 

### ISSUE THREAD

**Issue by huochunpeng** [cherow/cherow#254]
**Code**
I found this issue when parsing `i18next/dist/es/defaults.js`.

Simplified reproducible error:
```js
cherow.parseModule('export {get}; function get() {};');
```

**Expected behavior:**
Define a function named after "get" is fine in either browser or nodejs environment.
Define an object with a property "get" is fine in either browser or nodejs environment.
The parser should allow it.
As a reference, esprima can parse the above code.

**Actual behavior:**
It throws `SyntaxError: Line 1, column 8: Unexpected reserved word` with cherow v1.6.8.

**Comment by KFlash** [comment]
Thanx for this info. This will be fixed in v. 2.0. We don't do changes to 1.6.8 anymore because we focus on finalizing.

**Comment by fkleuver** [comment]
@KFlash If it's possible to roll out a fix for this (even if it's just quick-n-dirty) that would be preferred to pushing this to 2.0, since 2.0 will probably take a few weeks/months. This is blocking @huochunpeng from using it for critical use cases in a bundler he's working on. Please reconsider.

**Comment by KFlash** [comment]
If I fix that bug what will the version become? I can try looking into it
soon......

No timeframe on when 2.0 is done. Scoping and lexical bindings plus
bindings in general will take most of my time. I'm soon done with scoping...

**Comment by fkleuver** [comment]
1.6.9 (bugfix = patch bump)

**Comment by KFlash** [comment]
@huochunpeng A fix for this exist now. Waiting for @fkleuver to bundle it and push to NPM

**Comment by huochunpeng** [comment]
Thx! Guys.

**Comment by KFlash** [comment]
@huochunpeng I'm not sure why @fkleuver haven't merged and published this yet, but just wait for him.

**Comment by huochunpeng** [comment]
nvm, holiday mode I guess :)

**Comment by fkleuver** [comment]
@KFlash Work, man :) I'm not retired yet :)

Published now.

**Comment by huochunpeng** [comment]
It worked.

**Comment by KFlash** [comment]
@huochunpeng @fkleuver Just let you guys know that this is also fixed in upcoming 2.0, but the logic is very different.

In the example given by @huochunpeng `cherow.parseModule('export {get}; function get() {};');` it will throw an error if the function name is invalid because export will only throw if there is no exported binding.

Examples:

```js
export {get}; function get() {}; // Valid
export {get}; function set() {}; // Invalid - Will throw on undeclared exported binding
```

This is all after the ECMA specs, but Acorn fail short here.
